test/integration/ext_deletion_test.go

```diff
@@ -0,0 +1,100 @@
+// +build integration,etcd
+
+package integration
+
+import (
+	"testing"
+	"time"
+
+	newproject "github.com/openshift/origin/pkg/cmd/admin/project"
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	testutil "github.com/openshift/origin/test/util"
+	testserver "github.com/openshift/origin/test/util/server"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/errors"
+	expapi "k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/util/wait"
+)
+
+const projName = "ext-proj"
+
+func init() {
+	testutil.RequireEtcd()
+}
+
+func TestExtDeletion(t *testing.T) {
+	_, clusterAdminKubeConfig, err := testserver.StartTestMaster()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	clusterAdminClient, err := testutil.GetClusterAdminClient(clusterAdminKubeConfig)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	clusterAdminKubeClient, err := testutil.GetClusterAdminKubeClient(clusterAdminKubeConfig)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	// create the containing project
```

Comment:
Ah, ok.  I saw this way of doing it in one of the other tests.  `CreateNewProject` seems better here, though